Makefile

```diff
@@ -204,9 +210,9 @@ UIPL=$(ULROT)/ipl
 UPLUGINS=$(ULROT)/plugins/lib
 INST=$(SHTOOL) install -c
 F=*.{u,icn}
-Tbins=unicon icont iconx iconc unicont uniconx uniconc udb uprof unidep unidoc \
-	ui ivib patchstr iyacc rt.a rt.h
-
+Tbins=unicon$(EXE) icont$(EXE) iconx$(EXE) iconc$(EXE) unicont$(EXE) uniconx$(EXE) \
+      uniconc$(EXE) udb$(EXE) uprof$(EXE) unidep$(EXE) unidoc$(EXE) ui$(EXE) ivib$(EXE) \
+      patchstr$(EXE) iyacc$(EXE) rt.a rt.h
```

Comment:
While we are at it, can you please fix a few of those names to their parametrized one. see line 103 to 109 above.  Also, on Windows these are missing the windows-specific binaries such as wiconx.exe and wicont.exe, but the variables 103 to 109 should cover those too if you add them all here.

Reply:
you cab ignore this one. Just realized we have written this so we can handle iconx/uniconx renaming already. One thing to add if you want to make the `make install`  "complete" on windows, is to add window-specific binaries such as `wiconx` `wicont`, `wunicon`, etc, installed too.

Reply:
Notice that the you don't have to make that a windows conditional install. The install rule is smart enough to skip the ones that don't exist.